addons/medical/XEH_preInit.sqf

```diff
@@ -41,6 +41,13 @@ private _fixStatic = {
 };
 ["StaticWeapon", "init", _fixStatic] call CBA_fnc_addClassEventHandler;
 ["Car", "init", _fixStatic] call CBA_fnc_addClassEventHandler;
+["CAManBase", "Init", FUNC(handleInit)] call CBA_fnc_addClassEventHandler;
+["CAManBase", "Respawn", FUNC(handleRespawn)] call CBA_fnc_addClassEventHandler;
+["CAManBase", "Killed", FUNC(handleKilled)] call CBA_fnc_addClassEventHandler;
+["CAManBase", "Local", FUNC(handleLocal)] call CBA_fnc_addClassEventHandler;
+
+["ACE_bodyBagObject", "init", EFUNC(dragging,initObject)] call CBA_fnc_addClassEventHandler;
```

Comment:
dragging does 
```
    class ThingX {
        class ADDON {
            init = QUOTE(_this call DFUNC(initObject));
```
and `"ACE_bodyBagObject" isKindOf "ThingX" = true`
so I think we can drop this

Reply:
OK

Reply:
I didn't see a double init of dragging when profiling.. Maybe I just didn't look close enough though.